### ISSUE TITLE
Blurb, Text Module :: Line Height Value :: Undo option

### ISSUE THREAD

**Issue by eduard-ungureanu** [elegantthemes/Divi-Beta#520]
### Problem:

Let say I will customize the line height of the blurb module instead of the default 1.7em to be 25px, after saving I want to try a different line-height which: 35px, whiout closing the blurb module, I use the undo button - which should reset to the previous value which was 25px, instead it will reset to 2, then clicking again the same button will reset to the proper values 25px.

### Steps To Reproduce:
1.  add a blurb module
2. set the Body line-height to be 25px, save the module
3. edit again the module, and set the body line-height to be 35px
4. use the undo button

### Additional Information:

*Please include any of the following details that are relevant to the issue you are reporting.*

**Device**: macbook pro

**Operating System**: os siera

**Browser**: chrome

**Screen Resolution**:

**Screenshot or Screencast**:
https://drive.google.com/file/d/0B8PjD3Z_XQvuMWl2RkpkcXZIdm8/view?usp=sharing

**Comment by kary4** [comment]
I can confirm the issue, though on my end it works even strangely: http://youtu.be/EKB-2vjiCBs?hd=1

**Comment by SlavaET** [comment]
@kary4 you're seeing a different issue, it's not the issue Eduard described. Please check this - https://github.com/elegantthemes/submodule-builder/issues/355
It was fixed in the Visual Builder, but this issue still exist in Backend Builder.

**Comment by SlavaET** [comment]
Fixed. Actually I improved throttling of subhistory saving for the Range, color pickers and text which should be throttled. We already have the throttling there but it was working as follows:

1. Save the 1st value user input
2. Throttle for 1 sec to save the subsequent values.

So if you type the `12345` into input field, then history will be like this:

1. `initial state`
2. `1`
3. `12345`

Therefore reverting the changes looks a little weird in some cases.
With my fix throttling works from the beginning, i.e. it doesn't save the 1st value immediately and will work like this:

1. `initial state`
2. `12345`

Of course if you'll type too slow it'll save some middle values
